### ISSUE TITLE
HTML docs for action help. (Whatever st2 run blah.blah -h spits out)

### ISSUE THREAD

**Issue by lakshmi-kannan** [StackStorm/st2#1316]

**Comment by Kami** [comment]
I agree, I believe we should have help / reference docs for standard CLI commands in our documentation. We could and should automatically generate those docs (shouldn't be hard).

We should also generate help for all the core actions (so `st2 run core.<name>`) commands.

We did this for cast and it worked fine (see https://github.com/cloudkick/cast-site/blob/master/content/docs/client/bundles/delete/doc.stub for an example).
